src/authutil.ts

```diff
@@ -25,6 +27,16 @@ export function configAuthentication(
   writeFeedToFile(feedUrl, existingNuGetConfig, tempNuGetConfig);
 }
 
+function getExistingNugetConfig(processRoot: string) {
+  const configFileNames = fs
+    .readdirSync(processRoot)
+    .filter(filename => filename.toLowerCase() == 'nuget.config');

```

Comment:
Should it be `===`?

Reply:
`'nuget.config'` can be a variable like `defaultConfigName`

Reply:
Fixed, thanks!